client/src/pages/room.jsx

```diff
@@ -0,0 +1,252 @@
+import { useContext, useEffect, useState } from 'react';
+import { Context } from '../context/context.jsx';
+import { Formik, Form, Field } from 'formik';
+import { userService } from '../services/userService.js';
+import { useNavigate, useParams } from 'react-router-dom';
+import { ChatBlock } from './ChatsBlock.jsx';
+
+function messageValidation(newMessage) {
+  if (newMessage.trim().length > 300) {
+    return 'Message is to long';
```

Comment:
There is a typo in the error message: 'Message is to long' should be 'Message is too long'.